boards/x86/qemu_x86/doc/board.rst

```diff
@@ -63,3 +62,35 @@ The following platform features are unsupported:
 * Inter-Integrated Circuit (I2C)
 * Ethernet
 * Supervisor Mode Execution Protection (SMEP)
+
+Programming and Debugging
+*************************
+
+Use this board configuration to run basic Zephyr applications and kernel tests
```

Comment:
Use the `BOARD=qemu_x86` configuration to run basic Zephyr applications
     and kernel tests in the QEMU emulated environment, for example, with the
     :ref:`synchronization_sample`: